README.org

```diff
@@ -87,6 +87,7 @@ project will contain:
 - =+garden= sets up [[https://github.com/noprompt/garden][Garden]] and integrates into the build process
 - =+sass= sets up [[http://sass-lang.com][Sass]] and integrates into the build process (requires [[http://libsass.org][libsass]])
 - =+less= sets up [[http://lesscss.org/][Less]] and integrates into the build process.
+- =+test= sets up [[https://github.com/crisptrutski/boot-cljs-test][Less]] and adds a =test= task.
```

Comment:
there's still a `[less]` in here. not sure if that's shown or the alt text but probably good to fix in any case.